R/kernelshap.R

```diff
@@ -119,6 +119,7 @@
 #'   \item \code{prop_exact}: Proportion of the Kernel SHAP weight distribution covered by exact calculations.
 #'   \item \code{exact}: Logical flag indicating whether calculations are exact or not.
 #'   \item \code{txt}: Summary text.
+#'   \item \code{predictions}: (n x K) matrix of predictions of \code{X}.
```

Comment:
Maybe `A (n x K) matrix of predictions of \code{X}.` to keep these points in Sentence case

Reply:
Done, thx